tools/start_server.py

```diff
@@ -50,6 +50,8 @@
 
 class Flags():
   """Used to define global properties"""
+  HOST = '127.0.0.1'
```

Comment:
since these aren't global properties and used in other places like all other flags, i don't think these should be added to this class. I think we can just put the values in the default= expression without declaring them elsewhere.

Reply:
ADded in commit 30ac327